test/implementations/erc20pay.ts

```diff
@@ -64,10 +67,31 @@ async function equippableFixture(): Promise<Contract> {
     'MR',
     'ipfs://collection-meta',
     'ipfs://tokenURI',
-    [erc20.address, false, ADDRESS_ZERO, 0, 10000, ONE_ETH],
+    [erc20.address, isTokenUriEnumerated, lazyMintingEnabled, ADDRESS_ZERO, 0, 10000, ONE_ETH],
   ]);
 }
 
+async function lazyMintingDisabledFixture(): Promise<{
+  token: RMRKMultiAssetImplErc20Pay;
+  erc20: ERC20Mock;
+}> {
+  const erc20Factory = await ethers.getContractFactory('ERC20Mock');
+  const erc20 = <ERC20Mock>await erc20Factory.deploy();
+  await erc20.deployed();
+
+  const lazyMintingEnabled = false; // Shadows module lazyMintingEnabled
+  const token = <RMRKMultiAssetImplErc20Pay>(
+    await singleFixtureWithArgs('RMRKMultiAssetImplErc20Pay', [
+      'MultiAsset',
+      'MR',
```

Comment:
This is probably a remainder from the MultiResource era, maybe change it to `MA`? 😅 

```suggestion
      'MA',
```